intel-retail/core-services
# Python capi bindings

## PR Checklist


- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/core-services/blob/main/CONTRIBUTING.md

## What are you changing?

Adding python bindings for c-capi. This includes the docker image build to compile to .so file and an example python program to get you started.

## Issue this PR will close

close: #7 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- make build ovms-build-image
- make build-capi-python
- make run
- cd /tmp/project
- cp /output/ovmspybind.so .
- python3 ovms-pybind.py
- See isLive status True in a loop

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/core-services )
